docs/how-to/read-data-from-ioc.rst

```diff
@@ -0,0 +1,31 @@
+Read data from an IOC
+======================
+
+This guide explains how to read data from an IOC in a separate Python program. 
+
+.. note:: 
+    Please ensure your firewall allows both TCP and UDP traffic on ports 5064 and 5065. 
+    These are used by EPICS for channel access to the PVs.
+
+
+To start, run the :mod:`cothread` IOC from :doc:`../tutorials/creating-an-ioc` or the 
+:mod:`asyncio` IOC from :doc:`use-asyncio-in-an-ioc` and leave it running at the 
+interactive shell.
+
+We will read data from that IOC using this script:
+
+.. literalinclude:: ../examples/example_read_from_ioc.py
+
+.. note::
+    You may see warnings regarding the missing "caRepeater" program. This is an EPICS tool
+    that is used to track when PVs start and stop. It is not required for this simple example,
+    and so the warning can be ignored.
```

Comment:
According to https://github.com/dls-controls/aioca/pull/12#issuecomment-868862584 then if you upgrade the epicscorelibs dependency to `epicscorelibs>=7.0.5.99.1.0a2` (in pyproject.toml) then remove the Pipfile.lock and run `pipenv --rm && pipenv install --dev`` then these warnings should go away. Please could you try that?

Reply:
Doing this update causes tests to fail due to a missing import. I also see the same error when just trying to start up my example IOCs:
`  File "docs/examples/example_cothread_ioc.py", line 2, in <module>
    from softioc import softioc, builder
ModuleNotFoundError: No module named 'softioc'`

I can verify that this is entirely caused by changing the epicscorelibs dependency - swapping the required version back and reinstalling the pipenv makes the tests (and my example IOCs) work again.
I can't say I understand this error at the moment, so I'll investigate it a little more.

Reply:
Oops, you need to do `dls-py3 download-one-dependency epicscorelibs 7.0.5.99.1.0a2` first